frontend/src/features/public-form/PublicFormService.ts

```diff
@@ -93,6 +93,12 @@ export type SubmitStorageFormArgs = SubmitEmailFormArgs & {
   payments?: PaymentFieldsDto
 }
 
+export type SubmitStorageFormV2Args = SubmitEmailFormArgs & {
```

Comment:
suggestion: `SubmitStorageFormClearArgs`, to reduce the need to know the v1 vs v2 context

Reply:
Renamed in [7378d03](https://github.com/opengovsg/FormSG/pull/6665/commits/7378d0353be0e8ee013d4429d87b4126d216cd25) 👍

Reply:
Renamed more "V2"s to "Clear" in [d6e9db3](https://github.com/opengovsg/FormSG/pull/6665/commits/d6e9db359c1e494a792b28388408643b8bf8eb45)